<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_percent_export_data</class>
 <widget class="QDialog" name="dlg_percent_export_data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_read">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>百分比导出文件</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="m_lbName_2">
          <property name="text">
           <string>存储版文件路径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_leName_2"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="m_lbLocalPath_2">
          <property name="text">
           <string>本地文件路径：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_leLocalPath_2"/>
        </item>
        <item>
         <widget class="QPushButton" name="m_btnExport_2">
          <property name="text">
           <string>选择本地文件</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;百分比：(&lt;span style=&quot; font-size:10pt;&quot;&gt;%&lt;/span&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" rowspan="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="PercentLine"/>
        </item>
        <item>
         <widget class="QLineEdit" name="Rongliangline"/>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>导出容量：(MB)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QComboBox" name="m_cboxExportType_2"/>
        </item>
        <item>
         <widget class="QPushButton" name="pbtn_read_2">
          <property name="text">
           <string>开始导出</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
